lib/tasks/change_subscription_titles.rake

```diff
@@ -0,0 +1,29 @@
+namespace :subscriber_list do
+  desc "Update subscription titles to be more accurate and user friendly"
+  task change_titles: :environment do
+    puts "About to change titles"
+    ActiveRecord::Base.transaction do
+      translations = {
+          "policy-papers-and-consultations-with-3-document-types-2" => "Policy papers",
+          "policy-papers-and-consultations-with-4-document-types" => "Policy papers and open consultations",
+          "policy-papers-and-consultations-with-6-document-types" => "Policy papers, open consultations and closed consultations",
+          "policy-papers-and-consultations" => "Policy papers and consultations",
+          "policy-papers-and-consultations-with-3-document-types" => "Open consultations and closed consultations",
+          "policy-papers-and-consultations-with-2-document-types" => "Closed consultations",
+          "policy-papers-and-consultations-with-5-document-types" => "Policy papers and closed consultations",
+          "policy-papers-and-consultations-with-1-document-type" => "Open consultations"
```

Comment:
Is there a way to verify that these IDs are correct on every environment?

These may be correct for one environment, but incorrect for another.

For example `policy-papers-and-consultations-with-3-document-types` could be either "Policy papers and consultations" and "Policy papers" depending which was created first.

Reply:
Those were pulled from production and as the databases copied over nightly, this _shouldn't_ be a problem

Reply:
Awesome